src/components/DetailList.jsx

```diff
@@ -0,0 +1,42 @@
+import { Link } from 'react-router-dom'
+import '../styling/DetailList.css'
+
+export const DetailList = ({ detailData }) => {
+	console.log('check data in DetailList', detailData)
+	return (
+		<>
+			{/* {detailData &&
+				detailData.results.map((detail) => ( */}
+			<div className="detail-wrapper">
+				<div
+					className="detail-background"
+					style={{
+						backgroundImage: `url(https://media.themoviedb.org/t/p/w780${detailData.backdrop_path})`,
```

Comment:
Since there are screens that are bigger than 780; maybe next time you could consider on taking 1280px for background images.
-Izabel&Etna